src/coreclr/vm/stackwalk.cpp

```diff
@@ -2455,6 +2455,13 @@ StackWalkAction StackFrameIterator::NextRaw(void)
             if(isCollectableMethod)
                 fInsertCacheEntry = FALSE;
 
+            // Security stackwalk cache assumes it will not contain chained unwind info, so do not insert it.
+            UNWIND_INFO * pUnwindInfo = m_crawl.GetCodeInfo()->GetUnwindInfo();
```

Comment:
This won't work on non-x64. `UNWIND_INFO` and `UNW_FLAG_CHAININFO` are x64 specific structures. It would be nice to keep all special casing for this in EECodeInfo and avoid platform specific code here.